viper/parser/parser.py

```diff
@@ -62,9 +62,7 @@
 )
 
 
-try:
-    x = ast.AnnAssign
-except:
+if not hasattr(ast, 'AnnAssign'):
     raise Exception("Requires python 3.6 or higher for annotation support")
 
```

Comment:
Love this!

Reply:
It's so strange that flake8 suddenly decided to pick it up! :laughing: